Engine/platform/ios/acplios.cpp

```diff
@@ -359,15 +356,13 @@ void AGSIOS::DisplayAlert(const char *text, ...) {
   va_start(ap, text);
   vsnprintf(displbuf, 2000, text, ap);
   va_end(ap);
-  printf("%s", displbuf);
-//  ios_show_message_box(displbuf);
   
-  //while (ios_wait_for_ui)
-  //  usleep(200);
+  Debug::Printf(kDbgMsg_Warn, displbuf);
+  SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_WARNING, nullptr, displbuf, nullptr);
 }
 
 void AGSIOS::Delay(int millis) {
-  usleep(millis);
```

Comment:
I'm pretty sure this was incorrect because `usleep` accepts seconds and not milliseconds, so a conversion was needed. The SDL Implementation uses the best option for the platform and takes milliseconds as the input.